cluster/addons/calico-policy-controller/typha-horizontal-autoscaler-clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: typha-horizontal-autoscaler
```

Comment:
I like this naming, but we've called the other autoscaler resources `calico-cpva` for cluster-proportional-vertical-autoscaler.

Wondering if for consistency we should call these files / resources `typha-cpha` for "cluster-proportional-horizontal-autoscaler" - WDYT?

Reply:
Naming is hard :) I'll switch them to `typha-cpha`